import io.netty.util.concurrent.DefaultPromise.StacklessCancellationException;
public class FunctionUse22939 {
public void funcUse() {
StacklessCancellationException stacklesscancellationexception = new StacklessCancellationException();
stacklesscancellationexception.fillInStackTrace();
}
}